modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -309,6 +309,21 @@ mule -M-Dmule.deployment.parallel
 
 After performing these steps, Mule deploys in parallel all applications in the `/apps` directory.
 
+== Undeploying Applications
+
+You can undeploy Mule applications by deleting an app's anchor file only instead of deleting the application folder directly.
+
+Deleting only the app’s anchor file:
+
+* Prevents any interference from the hot-deployment layer and allows no room for concurrent conflicting actions.
+
+* Avoids potential application jar locking issues on some operation systems and allows for clean shutdown and undeployment.
+
+For example, if the stockTrader app is running (app folder is there as well as the $MULE_HOME/apps/stockTrader-anchor.txt file), just delete the anchor file to have the app removed from the Mule instance at runtime. Application folder is removed after the app terminates.
```

Comment:
what's the stockTrader app?

Reply:
`stockTrader` is just an app name the devs used to give an example. They said it's no relevant.